src/dbt_score/formatters/manifest_formatter.py

```diff
@@ -15,19 +15,19 @@ class ManifestFormatter(Formatter):
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         """Instantiate a manifest formatter."""
-        self._model_scores: dict[str, Score] = {}
+        self._evaluable_scores: dict[str, Score] = {}
         super().__init__(*args, **kwargs)
 
-    def model_evaluated(
-        self, model: Model, results: ModelResultsType, score: Score
+    def evaluable_evaluated(
+        self, evaluable: Evaluable, results: EvaluableResultsType, score: Score
     ) -> None:
-        """Callback when a model has been evaluated."""
-        self._model_scores[model.unique_id] = score
+        """Callback when an evaluable item has been evaluated."""
+        self._evaluable_scores[evaluable.unique_id] = score
 
     def project_evaluated(self, score: Score) -> None:
         """Callback when a project has been evaluated."""
         manifest = copy.copy(self._manifest_loader.raw_manifest)
-        for model_id, model_score in self._model_scores.items():
-            manifest["nodes"][model_id]["meta"]["score"] = model_score.value
-            manifest["nodes"][model_id]["meta"]["badge"] = model_score.badge
+        for evaluable_id, evaluable_score in self._evaluable_scores.items():
+            manifest["nodes"][evaluable_id]["meta"]["score"] = evaluable_score.value
```

Comment:
Noticing after taking another pass at the diff that sources need to be pushed into `manifest["sources"]` instead of `manifest["nodes"]`.  Let me fix that quickly.

Reply:
Oh good one, we should have a test for that

Reply:
updated in cc6b707